omnibus/config/templates/datadog-dogstatsd/upstart.conf.erb

```diff
@@ -0,0 +1,14 @@
+description "Datadog statsd server"
+
+start on started networking
+stop on runlevel [!2345]
+
+respawn
+
+console log  # redirect daemon's outputs to `/var/log/upstart/dogstatsd.log`
```

Comment:
To avoid duplicating logs in `upstart/dogstatsd.log` to `datadog/<log_file>` we have the `DD_LOG_TO_CONSOLE` option. It keeps all the logs in `datadog/<log_file>` but catch the panics in `upstart/dogstatsd.log`: https://github.com/DataDog/datadog-agent/blob/64499e4ee5ec95741392dda26103e7d074b17cd5/omnibus/config/templates/datadog-agent/upstart_debian.conf.erb#L12